en/page-breaking.saty

```diff
@@ -0,0 +1,288 @@
+      header-content = \metasub(${1}){bb}\;
+      header-origin = (\metasub(${1}){x}, \metasub(${1}){y})\;
+      \new-line;\quad;
+      footer-content = \metasub(${2}){bb}\;
+      footer-origin = (\metasub(${2}){x}, \metasub(${2}){y})\;
+      \new-line;
+      \|)
+    }%
+    Then, the function returns record formated above.
+    The record has information shows below to page-breaking processing system.
+
+    \listing{
+      * Draw sequence of block-boxs ${bb_1} as header, it pointed ${\tuple{|x_1|y_1|}} as left-top.
+      * Draw sequence of block-boxs ${bb_2} as header, it pointed ${\tuple{|x_2|y_2|}} as left-top.
+    }%
+    Please take care of axis of value. It is as same as 2nd argument that described previous paragraph;
+    this axis has coordinate origin is left-top and direction ${+y} increase page bottom.
+    Only \codem{page-break}'s 2nd and 3rd arguments use this axis.
+  }
+  +p{
+    Above paragraphs are all specifications of primitive \codem{page-break}.
+    Let me appends a bit to them, it is about \it{At time line-breaking actually}.
+，  Correctly, \it{\codem{page-break} itself processes page-breaking} is not true,
+    but starting process page-breaking  at all of document being returned value
+    as type  \type(Type.document);
+    In other words, a value as type \type(Type.document); is
+    \it{frozen} information about finally page-breaking that is not being processed page-breaking actually. 
+    It means; there is no process doing page-break actually written having return-value as type  
+    \type(Type.document); without last one. 
+    Because there is no primitive arranging type \type(Type.document);, 
+    There is no reason to create a value as type \type(Type.document); by \codem{page-break} 
+    without returning contents to output PDF.
+    The reason was what causing \it{side-effect} at page-breaking not using return-value,
+    and you can not do it\footnote{
+      There is no reason you cannot do it.
+      If needed, it is sounds good to change the specification what imcompatible in future;
+      \it{\codem{page-break} itself processes a page-breaking}.}.}
+
+  >
++section{primitves related page-breaking}<
+    +p{
+      There are some primitives to process related page-breaking.
+      Let me introduce them here.
+
+      \val-explain?:(`clear-page`)({clear-page})%
+        (Type.(block-boxes)) {
+          Page break in force.
+          In case inserted it at a target page unbreakable, 
+          it is equivalent as 
+          \codem{block-nil}.
+        }%
+
+      \val-explain?:(`add-footnote`)({add-footnote})%
+        (Type.(block-boxes --> inline-boxes)) {
+          
+          \codem{add-footnote \meta{bb}} inserts block-boxes \codem{\meta{bb}}
+          as footnote.
+          Return value has type \type(Type.inline-boxes);.
+          Associate with a place inserted this return value, 
+          and calculate inserting the footnote at processing page-breaking.  
```

Comment:
Insert subjects (it), or add -s to the verbs.

Reply:
mod the line.

 Associated with a place inserted this return value, 
          the footnote position is calculated at the page-breaking process.